js/api.js

```diff
@@ -15,20 +15,20 @@ const ErrorText = {
   SEND: 'Ошибка загрузки файла',
 };
 
-const request = async (url, method = HttpMethod.GET, body = null) => {
+const getPhotos = async (url, method = HttpMethod.GET, body = null) => {
```

Comment:
```suggestion
const responce = async (url, method = HttpMethod.GET, body = null) => {
```
Это неочевидно, но ответы сервера обычно так называют. Назване всё равно в этой функциии и дальше не идёт